draft-ietf-rats-architecture.md

```diff
@@ -1213,31 +1216,29 @@ For a discussion on the security of handles see {{handles-sec}}.
 The conveyance of Evidence and the resulting Attestation Results
 reveal a great deal of information about the internal state of a
 device as well as potentially any users of the device.
-In many cases, the whole point of the Attestation process is
+In many cases, the whole point of attestation procedures is
 to provide reliable information about the type of the device and the
 firmware/software that the device is running.
 This information might be particularly interesting to many attackers.
 For example, knowing that a device is
 running a weak version of firmware provides a way to aim attacks better.
 
-Many claims in Attestation Evidence and Attestation Results are potentially
-Personally Identifying Information) depending on the end-to-end use case of
-the attestation.
-Attestation that goes up to include containers and applications may further
-reveal details about a specific system or user.
+Many Claims in attestation Evidence and Attestation Results are potentially
+Personally Identifying Information (PII) depending on the end-to-end use case of
+the remote attestation procedure.
+Remote attestation that goes up to include containers and applications, e.g., in a multi-tenant architecture, may further
```

Comment:
The "multi-tenant architecture" example isn't wrong, but is different than the original case in mind.
The original case was that if you identify specific apps (e.g., a blood pressure monitor, etc.) the app id can be PII.